src/components/TimeBasedLineChart.js

```diff
@@ -553,7 +559,7 @@ const TimeBasedLineChart = React.createClass({
                   xScaleValueFunction={this._getXScaleValue}
                 />
               ) : null}
-              {alwaysShowZeroLine ? (
+              {alwaysShowZeroLine && this._yRangeContainsZero() ? (
```

Comment:
alwaysShowZeroLine implies to me if it is true the zero line should always be displayed, but there is a second check here in the logic -- can we rename alwaysShowZeroLine to something else, maybe just `showZeroLine`. I don't know, maybe there isn't a better option. But it does feel a little confusing here.